.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,6 +24,7 @@ This is for compliance purposes with FedRAMP program.
 ### Change Type
 
 <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
```

Comment:
Is this change required?

Reply:
It was added in the beta version so was added in this PR as well. Should we remove it?